Automattic/jetpack
# Jetpack plugin: always use s0.wp.com and i0.wp.com

For the vast majority of users, connections to `s[0-2].wp.com` and `i[0-2].wp.com` can be established using HTTP/2. When HTTP/2 (and in some cases even HTTP/1.1) is available, it's preferable to establish fewer connections to fewer origins, particularly when the number of resources to be retrieved is small.

As such, and for the vast majority of users, it makes sense to ensure that all connections are made to `s0.wp.com` and `i0.wp.com` rather than using hashes to pick among it and its other siblings.

This PR ensures that the `0` origin is used for every resource, and removes any existing preloading for `1` and `2`.

#### Changes proposed in this Pull Request:
* Always use `s0.wp.com` and `i0.wp.com` instead of picking among it and its other siblings
* Remove preloads for other origins

#### Jetpack product discussion
None.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Open a test page with comments, while logged into the site
* Ensure that masterbar-related resources load from `s0.wp.com` (e.g. `mustache` and `notes-common-v2`)
* Scan the document and ensure there are no prefetches for `s1.wp.com` or `s2.wp.com`
* Smoke-test masterbar and comment functionality to ensure nothing broke
